translations/monero-core_sv.ts

```diff
@@ -2604,7 +2604,7 @@ Index för spenderingsadress: </translation>
     <message>
         <location filename="../main.qml" line="862"/>
         <source>This address received %1 monero, with %2 confirmation(s).</source>
-        <translation>Denna adress tog emot %1 monero, med %2 bekräftelser.</translation>
+        <translation>Denna adress tog emot %1 Monero, med %2 bekräftelser.</translation>
```

Comment:
In Swedish we don't capitalize currency names. The Institute for Language and Folklore (Språkrådet) publishes a book called *Svenska Skrivregler*. Check out §§ 8.18 and 10.8.1 for example

Reply:
I got the impression that monero was capitalized on all other places, so this was just for consistency. However, I'm on vacation right now, so I can't check this right now.